.cirrus.yml

```diff
@@ -80,6 +80,10 @@ task:
       test_framework_script: |
         cd $FRAMEWORK_PATH/flutter/packages/flutter
         ../../bin/flutter test --local-engine=host_debug_unopt
+      # Run Flutter framework tests against the web engine.
+      test_framework_web_script: |
+        cd $FRAMEWORK_PATH/flutter/packages/flutter
```

Comment:
Shall we add this to LUCI instead since Godofredo is about to finish LUCI recipes :)

I think Linux Chrome already runs!

Reply:
Good point. I spoke with @godofredoc and for the time-being, we are continuing to use Cirrus until the end of Q1. So I think it's still worth adding these tests to Cirrus now.